tools/bin/commands/util/openshift_funcs

```diff
@@ -104,6 +104,31 @@ setup_oc() {
     exit 1
 }
 
+determine_console() {
+    local console=$(readopt --console)
+
+    if [ -z "${console}" ]; then
+        #
+        # 1. Show the status of oc
+        # 2. Cut the output to only the first line
+        # 3. Reduce the line to only the https ... url
+        #
+        # If logged out then console should be empty
+        #
+        console=$(oc status | head -n 1 | sed s/.*https/https/)
+    fi
+
+    if [[ ! ${console} =~ .*\/console ]]; then
```

Comment:
Please try to avoid bash constructs like `[[ ... ]]`. You can test this easily with parameter substitutions, too (something similiar like `${console//\/console$} != ${console}` or so ...

Reply:
I was going to but checked the rest of the script and noted that L234 uses the same kind of syntax hence thought this would suffice. Would you like L234 changed too?

Reply:
yeah, sorry. Sometimes I get a bit too dogmatic ;-) no problem, just wanted to mentionedn there are easier ways like e.g `[ ${console#/console} != $console ]` (the first expr removes the a trailing "/console"). That is POSIX compliant afaik.

sorr for the nit pick.